manifests/config/global.pp

```diff
@@ -16,6 +16,10 @@
   $split_key = split($name, '\.')
   $path = "/Users/${::boxen_user}/.gitconfig"
 
+  file { $path:
+    owner => $::boxen_user
```

Comment:
site.pp in boxen/our-boxen [defines a default `group` for file resources](https://github.com/boxen/our-boxen/blob/master/manifests/site.pp#L27-L30), so that is probably not an issue.

I would recommend being explicit with file modes, though.

The defaults do beg the question as to why `.gitconfig` was created with the wrong user but being explicit is a good idea.